reco_utils/recommender/sar/sar_singlenode.py

```diff
@@ -378,20 +421,23 @@ def get_item_based_topk(self, items, top_k=10, sort_top_k=False):
         # drop invalid items
         return df.replace(-np.inf, np.nan).dropna()
 
-    def recommend_k_items(self, test, top_k=10, sort_top_k=False, remove_seen=False):
+    def recommend_k_items(
+        self, test, top_k=10, sort_top_k=False, remove_seen=False, normalize=False
```

Comment:
Actually, `normalize` could be omitted from the arguments; it is not needed for computing top-k recommendations.

Reply:
I would argue for sorting the top-k results by default. Usually people want to know the predicted ranking.

Reply:
yep, makes sense